waspc/src/Generator.hs

```diff
@@ -1,27 +1,27 @@
 module Generator
        ( writeWebAppCode
-       , setup
+       , Generator.Setup.setup
```

Comment:
Here I am just reexporting these? Why? Because I am thinking of Generator.hs like of an interface toward the rest of the system. Although I am pretty sure there is stuff from Generator used outside of Generator and not via Generator.hs hm. We can remove this if you think it is silly.

Reply:
makes sense, I also think this is ok. If we look at it as one separate "thing", it would be strange to have it otherwise.